duckduckgo/iOS
# Use temp unprotected list and user list for all features

Task/Issue URL: https://app.asana.com/0/414709148257752/1200749768481992/f
Tech Design URL:
CC:

**Description**:
This PR checks the temp unprotected list and user unprotected list for all privacy features.

**Steps to test this PR**:
1. Visit https://privacy-test-pages.glitch.me/privacy-protections/fingerprinting/
2. Run the test `Include all headers and all properties of window, navigator and screen objects` checked
3. `screen.pixelDepth` should be 24
4. Turn off protections
5. Reload and run tests again
6. `screen.pixelDepth` should be 32 (on iPhone 13 sim)
7. Edit `ios-config.json` and add this to the `https` exceptions list:
```
                {
                    "domain": "good.third-party.site",
                    "reason": "sitebreakage"
                }
```
8. In `PrivacyConfigurationManager.swift` comment out lines 79-81 to force load the embedded config
9. Visit http://privacy-test-pages.glitch.me/privacy-protections/https-upgrades/
10. Run the test. Upgrade navigation should show an http url. NOT https.



**Device Testing**:


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
